rust/src/composepost.rs

```diff
@@ -1172,6 +1201,10 @@ OSTREE_VERSION='33.4'
             .symlink("var/lib/test/nested/symlink", "../")
             .unwrap();
 
+        // Also make this a sanity test for our directory size API
+        let cancellable = gio::Cancellable::new();
+        let _ = directory_size(rootfs.as_raw_fd(), cancellable.gobj_rewrap());
```

Comment:
Worth asserting `> 0` maybe?

Reply:
It turns out the total size is `42`.  Which is cool.